Components/input.html

```diff
@@ -83,52 +86,53 @@ <h1 class="documentation-component-main-heading">INPUT</h1>
                         <label for="text">
                             Name
                         </label>
-                        
+
                         <input type="text" name="text" autocomplete="name" placeholder="Enter name" />
                     </div>
-                    
+
                     <div class="input-label">
                         <label for="email">
                             Email *
                         </label>
 
                         <input type="email" name="email" required autocomplete="email" placeholder="Enter email" />
                     </div>
-                    
+
                     <div class="input-label">
                         <label for="password">Password *
                         </label>
 
-                        <input type="password" name="password" autocomplete="current-password" placeholder="Enter password" required value="..."/>
+                        <input type="password" name="password" autocomplete="current-password"
+                            placeholder="Enter password" required value="..." />
                         <label for="password">
                             wrong password, try again
                         </label>
                     </div>
-                    
+
                     <div class="input-label">
                         <label for="number">
                             Number
                         </label>
-                        
+
                         <input type="number" name="number" placeholder="Enter quantity" />
```

Comment:
you have provided "for" attribute to the label tag but it seems you forgot to add "id" attribute to the subsequent input tag. If you do that then input and label both are connected to each other and you get to focus on input after clicking on the related label tag.